content/java/concepts/array-list/terms/removeRange/removeRange.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: '.removeRange()'
+Description: 'Removes every element within a specified range.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.removeRange()` method is used for removing all elements inside a specified range from an instance of the ArrayList class.
```

Comment:
```suggestion
The `.removeRange()` method is used for removing all elements inside a specified range from an instance of the `ArrayList` class.
```
Link the codecademy's [ArrayList](https://www.codecademy.com/resources/docs/java/array-list) URL here.

Reply:
Didn't add the link as this entry is under the same concept.